airbyte-cdk/python/README.md

```diff
@@ -2,9 +2,9 @@
 
```

Comment:
That file's contents are also duplicated as a docs page. I wonder if there's a nice clean way to import a markdown file with a docusaurus-blessed imnport markdown syntax, from outside of it's content directory?